packages/engineer/src/feature/dev-server.dev-server.env.ts

```diff
@@ -70,8 +71,12 @@ devServerFeature.setup(
 
         run(async () => {
             // Should engine config be part of the dev experience of the engine????
-            const { require, socketServerOptions: configServerOptions, serveStatic } =
-                (await application.getEngineConfig()) ?? {};
+            const {
+                require,
+                socketServerOptions: configServerOptions,
+                serveStatic,
+                webpackConfigPath: definedWebpackConfigPath,
```

Comment:
this should not be loaded from engine config.

Reply:
why?